code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -155,30 +155,12 @@
 	S.DisIntegrate(src)
 	toggle_cam(S, 0)
 
-/obj/machinery/particle_accelerator/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+/obj/structure/particle_accelerator/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
```

Comment:
The reason was that the typepath was wrong, pointing to an object that wasn't actually the parent of the other objects. Still, any code ought to be tested. I'll have the results in a few minutes.